easybuild/easyblocks/o/openfoam.py

```diff
@@ -438,14 +438,14 @@ def sanity_check_step(self):
         if self.is_dot_org and self.looseversion >= LooseVersion('7'):
             tools.remove("buoyantBoussinesqSimpleFoam")
             tools.remove("sonicFoam")
-        # buoyantSimpleFoam replaced by buoyantFoam in versions 10+
-        if self.is_dot_org and self.looseversion >= LooseVersion("10"):
+        # engineFoam replaced by reactingFoam and buoyantSimpleFoam replaced by buoyantFoam in version 10
+        if self.is_dot_org and LooseVersion("10") <= self.looseversion:
             tools.remove("buoyantSimpleFoam")
-            tools.append("buoyantFoam")
-        # engineFoam replaced by reactingFoam in versions 10+
-        if self.is_dot_org and self.looseversion >= LooseVersion("10"):
             tools.remove("engineFoam")
-            tools.append("reactingFoam")
+            # both removed in version 11
+            if self.looseversion < LooseVersion("11"):
+                tools.append("buoyantFoam")
+                tools.append("reactingFoam")
```

Comment:
out of curiosity: isn't there another (replacement) solver we can check for instead with OpenFOAM 11+?